src/components/dataset-query/dataset-query.js

```diff
@@ -37,16 +43,41 @@ const DatasetQuery = props => {
   const updateLocalQueryText = e => setLocalQueryText(e.target.value)
   const updateReduxQueryText = (e) => setQueryText(e.target.value)
 
-  const textArea = <textarea rows={5} type='text' value={localQueryText} onBlur={updateReduxQueryText} onChange={updateLocalQueryText} className='query-input' />
+  const textArea = <textarea rows={5} type='text' value={localQueryText} onBlur={updateReduxQueryText} onChange={updateLocalQueryText} className='sql-textbox' />
   const submitButton = <button className="action-button" disabled={isQueryLoading} onClick={submit}>Submit</button>
   const cancelButton = <button className="action-button" disabled={!isQueryLoading} onClick={cancel}>Cancel</button>
-  
+
   const showSuccessMessage = !queryFailureMessage && userHasInteracted && !isQueryLoading
   const successMessage = showSuccessMessage && (
     <span className='success-message'>
       Query successful.  To refresh the visualization, you must change an element in the trace
     </span>
   )
+  const createHoverText = (systemName) => showTooltipCopied ? 'Copied!' : `Copy table name '${systemName}'`
+
+  const recommendationItems = () => {
```

Comment:
Is there a reason this can't be broken out into a component similar to the datasets recommendation component so it's not muddying up this component as much?

Reply:
Also, I trust you've asked this question before, but is it coupled enough yet with the query itself to have it here? Are there plans to make them more coupled?

Reply:
We're splitting it out. We decided to leave it in there until it warranted being it's own component and by the time that happened we sorta didn't notice.

Reply:
Are you asking if it makes sense to render it on the custom query tab?